lib/env.js

```diff
@@ -0,0 +1,13 @@
+const snapShot = require('snap-shot-core');
+
+/** @type {snapShot.SnapEnvOptions} */
+const OPTIONS_FROM_ENV = {
+  show: process.env.SNAPSHOT_SHOW === 'true',
+  dryRun: process.env.SNAPSHOT_DRY === 'true',
+  update: process.env.SNAPSHOT_UPDATE === 'true',
+  ci: process.env.SNAPSHOT_CI === 'true' || process.env.EMBER_TRY_SCENARIO
```

Comment:
I'm not 100% what `OPTIONS_FROM_ENV.ci` is used for ATM, but if its trying to see if we are "running in ci" I'd suggest using https://github.com/watson/ci-info (its what npm uses, and we are working to migrate to it in ember-cli as well).

Forgive the noise if that isn't the goal of `.ci` here though...

Reply:
Basically, the point is to ensure that new snapshots can _never_ be created in a CI environment. 7b4e303 replaces `EMBER_TRY_SCENARIO` with ci-info's `isCI`